src/algebra/star/basic.lean

```diff
@@ -7,6 +7,7 @@ import algebra.ring.aut
 import algebra.ring.comp_typeclasses
 import data.rat.cast
 import group_theory.group_action.opposite
+import group_theory.submonoid.basic
```

Comment:
I think it might make sense to move this content on ordering into a new `algebra/star/order` file.